README.md

```diff
@@ -284,30 +284,36 @@ Obtain overlap-aware speaker embeddings from a microphone stream:
 ```python
 import rx.operators as ops
 import diart.operators as dops
-from diart.sources import MicrophoneAudioSource
+from diart.sources import MicrophoneAudioSource #, FileAudioSource
 from diart.blocks import SpeakerSegmentation, OverlapAwareSpeakerEmbedding
 
-segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation")
-embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding")
+segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation", use_hf_token="")
+embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding", use_hf_token="")
+
 mic = MicrophoneAudioSource()
+# To take input from file:
+# mic = FileAudioSource("<filename>", sample_rate=16000)
 
 stream = mic.stream.pipe(
     # Reformat stream to 5s duration and 500ms shift
-    dops.rearrange_audio_stream(sample_rate=segmentation.model.sample_rate),
+    dops.rearrange_audio_stream(sample_rate=16000),
     ops.map(lambda wav: (wav, segmentation(wav))),
     ops.starmap(embedding)
-).subscribe(on_next=lambda emb: print(emb.shape))
+).subscribe(on_next=lambda emb: print(emb)) #emb.shape to display shape
```

Comment:
```suggestion
).subscribe(on_next=lambda emb: print(emb.shape))
```

I prefer to leave the shape here so the example output isn't polluted. Users will understand that removing `.shape` will print the actual embeddings